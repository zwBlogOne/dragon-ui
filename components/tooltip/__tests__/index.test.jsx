import React from 'react';
import { render } from 'enzyme';
import toJson from 'enzyme-to-json';
import Tooltip from '../index';
import Button from '../../button';

describe('Tooltip', () => {
  it('renders normal Tooltip correctly', () => {
    const wrapper = render(
      <Tooltip title="这是一个left定位的Tooltip" direction="left">
        <Button>left</Button>
      </Tooltip>
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});

