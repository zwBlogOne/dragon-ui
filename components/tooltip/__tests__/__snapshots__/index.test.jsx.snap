// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip renders normal Tooltip correctly 1`] = `
<div
  class="ui-popover ui-tooltip"
>
  <div
    class="ui-popover-content ui-popover-content-radius"
  >
    这是一个left定位的Tooltip
    <span
      class="ui-popover-arrow"
    />
  </div>
  <div>
    <button
      class="ui-button theme-default"
      type="button"
    >
      left
    </button>
  </div>
</div>
`;
